docs/source/examples.rst

```diff
@@ -93,11 +93,6 @@ Write time series
 .. literalinclude:: /../../src/volue/mesh/examples/write_timeseries_points.py
    :language: python
 
-Write time series, asynchronously
-*********************************
-.. literalinclude:: /../../src/volue/mesh/examples/write_timeseries_points_async.py
```

Comment:
I don't find the write_timeseries_points_async.py file - missing check-in?

Reply:
It was removed[^1] in the last PR #422.  In this PR I'm removing the reference to it from documentation's examples page.

[^1]: To be precise it was merged into one script: write_timeseries_points.py

Reply:
Sorry - my misreading 😢